images/20-zfs/dev.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
delete me - but think about what a user should do when they want to develop a new service